### ISSUE TITLE
Attempt to Decode deck somehow fails when running in Heroku

### ISSUE THREAD

**Issue by LiquidRekto** [Rafalonso/LoRDeckCodesPython#1]
Just recently, I used the code to run a deck info function for my Discord Bot. The bot is hosted via Heroku, and obviously the bot uses python to run. Decoding session was running fine and smooth.

However, when I deployed and tested the bot again today, somehow it triggered errors. Here's the log if you want to know further
`2020-04-26T13:58:57.656093+00:00 app[worker.1]: Ignoring exception in on_message`

`2020-04-26T13:58:57.657203+00:00 app[worker.1]: Traceback (most recent call last):`

`2020-04-26T13:58:57.657304+00:00 app[worker.1]:   File "/app/.heroku/python/lib/python3.8/site-packages/discord/client.py", line 312, in _run_event`

`2020-04-26T13:58:57.657305+00:00 app[worker.1]:     await coro(*args, **kwargs)`

`2020-04-26T13:58:57.657350+00:00 app[worker.1]:   File "bot.py", line 81, in on_message`

`2020-04-26T13:58:57.657351+00:00 app[worker.1]:     deck = LoRDeck.from_deckcode('CEBAIAIFB4WDANQIAEAQGDAUDAQSIJZUAIAQCBIFAEAQCBAA')`

`2020-04-26T13:58:57.657419+00:00 app[worker.1]:   File "/app/.heroku/python/lib/python3.8/site-packages/lor_deckcodes/models.py", line 34, in from_deckcode`

`2020-04-26T13:58:57.657420+00:00 app[worker.1]:     return cls(decode_deck(deckcode))`

`2020-04-26T13:58:57.657453+00:00 app[worker.1]:   File "/app/.heroku/python/lib/python3.8/site-packages/lor_deckcodes/decode.py", line 26, in decode_deck`

`2020-04-26T13:58:57.657454+00:00 app[worker.1]:     raise ValueError("Version/Format not supported.")`

`2020-04-26T13:58:57.657505+00:00 app[worker.1]: ValueError: Version/Format not supported.`

In short words, the problems rises directly in the decode function. I don't know what happened, but everything was running so smooth until this came in. (Is this might be a problem from the previous day's commit?).
Hope there will be responses to this issue.
P/s: If the code still works in Python console, then this is definitely weird

**Comment by Rafalonso** [comment]
Hello @LiquidRekto, 
Indeed the bug was introduced yesterday, I'll be issuing a fix today, thanks for bringing this to my attention, in the meanwhile if you must you can use the previous version by installing the version 1.0.0.

**Comment by LiquidRekto** [comment]
Just modified the dependency's version as you have said: `lor-deckcodes==1.0.0`
The program worked well again.
Thank you for the solution! Hope this will be fixed soon, since set 2 will be coming in no time!
P/s: You can close this issue if you want

**Comment by Rafalonso** [comment]
@LiquidRekto 
You can upgrade to version 1.0.2 now, the issue is fixed and version 2 of the decks with BilgeWater are now available for decoding.
Thank you for bringing this to my attention.
